nanoFramework.Device.Can/CanTiming.cs

```diff
@@ -0,0 +1,83 @@
+//
```

Comment:
file name must match the name of the class, so please rename to CanTimingSTM32.cs :-)